test-tools/src/main/java/io/neow3j/test/NeoExpressTestContainer.java

```diff
@@ -1,64 +1,192 @@
 package io.neow3j.test;
 
-import org.testcontainers.containers.Container;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
 import org.testcontainers.utility.DockerImageName;
 import org.testcontainers.utility.MountableFile;
 
-public class NeoExpressTestContainer extends GenericContainer<NeoExpressTestContainer> {
-
-    /**
-     * Default location for a neoxp checkpoint file in the container.
-     */
-    public static final String CHECKPOINT_DEST = "/app/setup.neoxp-checkpoint";
+import java.math.BigInteger;
 
-    /**
-     * Default location for a neoxp batch file in the container.
-     */
-    public static final String BATCH_DEST = "/app/setup.batch";
+public class NeoExpressTestContainer extends GenericContainer<NeoExpressTestContainer> {
```

Comment:
Can we make an interface from this as we discussed in a call? To enable extensibility in the future.

Then, the implementations could be: `NeoExpressTestDocker` and `NeoExpressTestLocal` or something like that. Wdyt?

Reply:
Yes, that makes sense. But do you think it is necessary to create the interface now? I prefer doing it only as soon as we have at least two implementations.